core/src/test/resources/sql/operator.iq

```diff
@@ -702,4 +702,54 @@ FOO BOOLEAN(1) NOT NULL
 
 !ok
 
+SELECT 2 ^ 3 AS foo;
```

Comment:
these tests are all tests with constants, please add some non-constant tests, e.g., using empno/empid.
Also, please mix operand types.
That's the whole point of iq test - they can run on table data. Better coverage than operator tests.